.circleci/config.yml

```diff
@@ -67,6 +68,10 @@ commands:
             RAILS_VERSION: <<parameters.rails_version>>
             SOLIDUS_BRANCH: <<parameters.solidus_branch>>
           when: always
+      - when:
+          condition: <<parameters.coverage>>
```

Comment:
I'm missing (even on Stripe) when this parameter will be `true`. In other words, who sets this?